### PR TITLE
Pytest fixtures.

### PR DESCRIPTION
**Pull request by jgoclawski**
This change is [Reviewable](https://reviewable.io/reviews/socialwifi/sqlalchemy-postgres-autocommit/4)

### CODE REVIEW

**Comment by jgoclawski** [comment]
@mimo95 please take a look.

File: setup.py [hunk 1]
```diff
@@ -14,6 +14,11 @@
     install_requires=[str(ir.req) for ir in parse_requirements('base_requirements.txt', session=False)],
     setup_requires=['pytest-runner'],
     tests_require=['pytest'],
+    entry_points = {
+        'pytest11': [
```
Comment (mimo00) [inline setup.py]: Why '*11' ?

File: sqlalchemy_postgres_autocommit/pytest_fixtures.py [hunk 2]
```diff
@@ -24,3 +24,23 @@ def db_session(autocommit_database, db_connection):
     session = autocommit_database.session_factory(bind=db_connection)
     yield session
     session.close()
+
+
```
Comment (mimo00) [inline sqlalchemy_postgres_autocommit/pytest_fixtures.py]: Fixtures should be in 'Add pytest fixtures for transactional tests.' commit not in Add documentation.

**Review by mimo00 — COMMENTED** [review]
Looks good, useful fixtures. The question is why to configure db_session/db_connection through other fixtures, not more explicite (dict, file, constans) ?

**Comment by mimo00** [comment]
@jgoclawski waiting for merge !

**Comment by jgoclawski** [comment]
@mimo00 I've updated the PR, please take a look. Configuration is now done with factories instead of fixtures. Readme is updated to reflect the change.

**Review by mimo00 — APPROVED** [review]
LGTM, like factory configuration !

**Pull request merged**